RELEASES.md

```diff
@@ -1,3 +1,78 @@
+Version 1.29.0 (2018-09-13)
+==========================
+
+Compiler
+--------
+- [Bumped minimum LLVM version to 5.0.][51899]
```

Comment:
Sorry if this is naive to ask -- but is this ever actually visible to users? Isn't LLVM vendored either way?

Reply:
It's relevant to distros, so worth noting here I think.

Reply:
Again, this is entirely me being uninformed, but if you have time do you mind explaining to me why it is relevant to distros? (If you don't have time or don't care to explain this I completely understand)

Reply:
Since we've removed code from the LLVM shims that was needed for 4.9 (or whatever the previous version was, maybe 3.9) compatibility, distros that package that previous LLVM might need to do more work now. However, I believe that someone did a loose survey and we believe that most if not all distros are on 5.0+ already.

Reply:
Isn't it vendored either way though?

Reply:
You can build with your own external LLVM too, and distros usually do.